src/main/java/carpet/patches/NetHandlerPlayServerFake.java

```diff
@@ -4,8 +4,11 @@
 import net.minecraft.network.chat.Component;
 import net.minecraft.network.chat.contents.TranslatableContents;
 import net.minecraft.network.protocol.Packet;
+import net.minecraft.network.protocol.game.*;
```

Comment:
please not star imports.

Reply:
Oops. IntelliJ did that, I had imported many things from there whilst investigating. My commit though, so my bad.